### ISSUE TITLE
Kedro viz not displaying properly with proxy

### ISSUE THREAD

**Issue by QuentinLetort** [kedro-org/kedro-viz#1627]
## Description
Kedro-viz is not displaying details of nodes when a proxy is set. The pipeline completely disappear when I click on a node.

## Context
I try to run kedro-viz behind a proxy in my company (Linux RedHat) but I'm not able to see node's detail (as soon as I click on a node, the pipeline disappear).
I recreate the exact same problem on my personal laptop (MacOS) using Jupyter Server Proxy and iris-pandas template. 

## Steps to Reproduce

1. Install jupyter-server-proxy and create a new Kedro project using iris-pandas template 
2. Add an entry in jupyter config (https://jupyter-server-proxy.readthedocs.io/en/latest/server-process.html#specifying-config-via-traitlets)
3. Launch Kedro-Viz from Jupyter Lab
<img width="240" alt="Capture d’écran 2023-11-07 à 15 47 08" src="https://github.com/kedro-org/kedro-viz/assets/43543191/ff0602b2-c55d-4c7a-8913-4f58c36b52da">

4. Click on a node

## Expected Result
Clicking on a node when Kedro-Viz is used behind a proxy should give the same behaviour as when Kedro-Viz is used locally

## Actual Result
Pipeline disappear or appear again after a while ~10s (but only details of the clicked node can be displayed)
 
```
-- If you received an error, place it here.
```

```
-- Separate them if you have more than one.
```

## Your Environment
Include as many relevant details as possible about the environment you experienced the bug in:

* Web browser system and version: Firefox and Edge
* Operating system and version: MacOS and Redhat 
* NodeJS version used (if relevant): 
* Kedro version used (if relevant): last version
* Python version used (if relevant): 3.9.7

## Checklist
- [x] Include labels so that we can categorise your issue

**Comment by QuentinLetort** [comment]
Downgrading the version to the one proposed in the #916 issue allow me to used Kedro-Viz behind the proxy (but I get other bugs solved since that version)

**Comment by tynandebold** [comment]
Thanks for the issue, @QuentinLetort. We've just fixed this problem with #1621 and will release a new version soon.

**Comment by QuentinLetort** [comment]
Thanks @tynandebold, I just tested it with the new release and it's working fine
